src/features/stats/reducer.js

```diff
@@ -34,6 +39,11 @@ const statsReducer = (state = initialState, { type, payload }) => {
             return { ...state, gold: state.gold - payload };
 
         case 'SET_ABILITY_SCORES':
+            const moreMana =
+                calculateMaxManapool(
+                    calculateModifier(payload.abilities.intelligence)
+                ) - state.maxMana;
+            state.mana += moreMana;
```

Comment:
This `moreMana` variable seems redundant.

Reply:
I kind of prefer it this way, it's nicer to look at, makes it more clear what the value being added is, and makes it simpler to change in future.

Reply:
I think your second point would only be true if a more descriptive variable name was used.

Reply:
You're right, but sort of missing the point. I think a more descriptive name would be very appreciated here.

Reply:
Actually, upon reflection, I think you're right in saying the variable should be kept. But it does certainly need a more descriptive name.

Reply:
This segment of code is used to increase your current mana by the amount gain from investing more points into the INT stat. I agree that the name could be more descriptive but I was following the established format for this which is increasing health by the amount gain on level up (Line 312-315 in the same file). Therefore, if we changed the name for this variable then I argue we should change it for hp as well.

Reply:
Oh, I didn't even realise that existed. That's a good point.

Reply:
Discussed: Leave as it is.